tests/waku_rln_relay/test_rln_group_manager_static.nim

```diff
@@ -137,18 +145,28 @@ suite "Static group manager":
     let fut = newFuture[void]()
 
     proc callback(registrations: seq[Membership]): Future[void] {.async.} =
-      require:
-        registrations.len == 1
-        registrations[0].idCommitment == idCommitment
-        registrations[0].index == 10
+      when defined(rln_v2):
+        require:
```

Comment:
nitpick. `registrations.len == 1` and `registrations[0].index == 10` is the same in v1 and v2. extract it and remove the duplicate?

Reply:
addressed in 320661dc